packages/react-experiments/src/components/SelectedItemsList/Items/ItemWithContextMenu.tsx

```diff
@@ -21,7 +21,7 @@ export const ItemWithContextMenu = <T extends any>(
       setIsContextMenuOpen(true);
     }, [setIsContextMenuOpen]);
     const closeContextMenu = React.useCallback(
-      e => {
+      (e: Event | React.MouseEvent | React.KeyboardEvent) => {
```

Comment:
wasn't this previously `any` ? I'd suggest to avoid doing any type restrictive additions to old codebase